src/sentry/api/paginator.py

```diff
@@ -233,7 +235,14 @@ def __init__(
         self.max_offset = max_offset
         self.on_results = on_results
 
-    def get_result(self, limit=100, cursor=None):
+    def get_result(
+        self,
+        limit: int = 100,
+        cursor: Any = None,
+        count_hits: Any = False,
+        known_hits: Any = None,
+        max_hits: Any = None,
```

Comment:
Downstream this is expected to conform to the `PaginatorLike` protocol, although the three new kwargs are ignored.